### PR TITLE
guess we mergin backend again

### PR DESCRIPTION
**Pull request by PoolloverNathan**

### CODE REVIEW

**Review by penguinencounter — CHANGES_REQUESTED** [review]
that is not how `__index` is supposed to work :sob:

**Comment by PoolloverNathan** [comment]
> that is not how `__index` is supposed to work 😭

true

File: common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java [hunk 1]
```diff
@@ -247,6 +247,11 @@ public boolean isTraceEnabled() {
     @LuaWhitelist
     public final BackendAPI backend = new BackendAPI();
 
+	@LuaWhitelist
+	public final BackendAPI __index() {
+		return backend;
```
[the remark this answers is not in the document]
Comment (TheBunnyMan123) [reply common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java:252]: what the actual fuck

File: common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java [hunk 2]
```diff
@@ -247,6 +247,12 @@ public boolean isTraceEnabled() {
     @LuaWhitelist
     public final BackendAPI backend = new BackendAPI();
 
+	@LuaWhitelist
+	public final BackendAPI __index(String s) {
+		if (s == "backend") return null;
```
Comment (penguinencounter) [inline common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java:252]: # :sob:

**Review by penguinencounter — CHANGES_REQUESTED** [review]

**Comment by PoolloverNathan** [comment]
Competitive GitHub

**Review by TheBunnyMan123 — APPROVED** [review]
TODO: Fix connect function

**Pull request merged**